test/mcmcchains.jl

```diff
@@ -0,0 +1,38 @@
+using ReTest, Random, AdvancedHMC, ForwardDiff, AbstractMCMC, MCMCChains
+using Statistics: mean
+include("common.jl")
+
+@testset "MCMCChains w/ gdemo" begin
+    rng = MersenneTwister(0)
+
+    n_samples = 5_000
+    n_adapts = 5_000
+
+    θ_init = randn(rng, 2)
+
+    model = AdvancedHMC.LogDensityModel(LogDensityProblemsAD.ADgradient(Val(:ForwardDiff), ℓπ_gdemo))
+    init_eps = Leapfrog(1e-3)
+    κ = NUTS(init_eps)
+    metric = DiagEuclideanMetric(2)
+    adaptor = StanHMCAdaptor(MassMatrixAdaptor(metric), StepSizeAdaptor(0.8, κ.τ.integrator))
+
+    samples = AbstractMCMC.sample(
```

Comment:
Can you add a test in which `bijector` is used?

Reply:
Done. I think I got a bit confused as to where bijectors belong, this is probably not the right place.